terraform/modules/baseline_presets/s3.tf

```diff
@@ -51,246 +51,22 @@ locals {
-        "s3:GetObjectTagging",
-        "s3:ListBucket",
-      ]
-      principals = {
-        type = "AWS"
-        identifiers = [for account_name in var.environment.prodpreprod_account_names :
-          var.environment.account_root_arns[account_name]
-        ]
-      }
-    }
-
-    ProdPreprodEnvironmentsWriteAccessBucketPolicy = {
-      effect = "Allow"
-      actions = [
-        "s3:GetBucketLocation",
-        "s3:GetObject",
-        "s3:GetObjectTagging",
-        "s3:ListBucket",
-        "s3:PutObject",
-        "s3:PutObjectAcl",
-        "s3:PutObjectTagging",
-        "s3:RestoreObject",
-      ]
-      principals = {
-        type = "AWS"
-        identifiers = [for account_name in var.environment.prodpreprod_account_names :
-          var.environment.account_root_arns[account_name]
-        ]
-      }
-    }
-
-    AllEnvironmentsWriteAndDeleteAccessBucketPolicy = {
-      effect = "Allow"
-      actions = [
-        "s3:GetBucketLocation",
-        "s3:GetObject",
-        "s3:GetObjectTagging",
-        "s3:ListBucket",
-        "s3:PutObject",
-        "s3:PutObjectAcl",
-        "s3:PutObjectTagging",
-        "s3:DeleteObject",
-        "s3:RestoreObject",
-      ]
-      principals = {
-        type = "AWS"
-        identifiers = [for account_name in var.environment.account_names :
-          var.environment.account_root_arns[account_name]
-        ]
-      }
-    }
-
-    DevTestEnvironmentsReadOnlyAccessBucketPolicy = {
-      effect = "Allow"
-      actions = [
-        "s3:GetBucketLocation",
-        "s3:GetObject",
-        "s3:GetObjectTagging",
-        "s3:ListBucket",
-      ]
-      principals = {
-        type = "AWS"
-        identifiers = [for account_name in var.environment.devtest_account_names :
-          var.environment.account_root_arns[account_name]
-        ]
-      }
-    }
-
-    DevTestEnvironmentsWriteAndDeleteAccessBucketPolicy = {
-      effect = "Allow"
-      actions = [
-        "s3:GetBucketLocation",
-        "s3:GetObject",
-        "s3:GetObjectTagging",
-        "s3:ListBucket",
-        "s3:PutObject",
-        "s3:PutObjectAcl",
-        "s3:PutObjectTagging",
-        "s3:DeleteObject",
-        "s3:RestoreObject",
-      ]
-      principals = {
-        type = "AWS"
-        identifiers = [for account_name in var.environment.devtest_account_names :
-          var.environment.account_root_arns[account_name]
-        ]
-      }
-    }
-
-    DevelopmentReadOnlyAccessBucketPolicy = {
-      effect = "Allow"
-      actions = [
-        "s3:GetBucketLocation",
-        "s3:GetObject",
-        "s3:GetObjectTagging",
-        "s3:ListBucket"
-      ]
-      principals = {
-        type = "AWS"
-        identifiers = [
-          var.environment.account_root_arns["${var.environment.application_name}-development"]
-        ]
-      }
-    }
+    ImageBuilderWriteAccessBucketPolicy                 = local.iam_policy_statements.S3ReadWriteCoreSharedServicesProduction[0]
```

Comment:
Not sure whether the Bucket Policies belong in the same place as the IAM ones. They are slightly different things

Reply:
how about 
s3 statements in the s3 file under s3_policy_statements?